Gruntfile.js

```diff
@@ -69,6 +70,18 @@ module.exports = exports = function(grunt) {
                         // just to add the BANNER
                         // without adding an extra grunt module
                         'dist/localforage.nopromises.js'
+                    ],
+                    // Node output files
+                    'dist/localforage.node.js': [
+                        'src/utils/node_banner.js',
+                        'bower_components/es6-promise/promise.js',
+                        'dist/localforage.nopromises.js'
+                    ],
+                    'dist/localforage.nopromises.node.js': [
+                        // just to add the BANNER
+                        // without adding an extra grunt module
+                        'src/utils/node_banner.js',
+                        'dist/localforage.nopromises.js'
```

Comment:
To replace the dist files (i.e. for getting the header in there), what would be a better approach;

* Making an `index.js` file for nodejs, which includes the node.js header and the rest of the library?